### ISSUE TITLE
Rethink session end date for semester count

### ISSUE THREAD

**Issue by ajabbot** [gordon-cs/gordon-360-ui#353]
Could we re-think our Session End Date so that the semester counter shows the Days Remaining until the last day of Finals (which academically is the "deadline" students most care about). Session End Dates showing Days Remaining beyond that pose a confusing or perceived-inaccurate days remaining count?

**Comment by EjPlatzer** [comment]
This is also an issue with term beginnings, especially when (on years like this academic year) the term beginning doesn't align as closely with the first day of classes as expected. 360 is getting these values from a view that ultimately depends on data in Jenzabar (see TmsEPrd view GORD_CCT_CM_SESSION_MSTR for more details). The fields WHEEL_BEGN_DTE and WHEEL_END_DTE have been added to this view, I think for this express purpose. Those fields are maintained separately from the College's actual begin and end date for terms. This way, someone can manually update them to reflect a more user-accurate definition of semester days transpired and remaining.

However, the API currently does not make use of these fields in determining the days transpired and days remaining.

**Comment by ChrisImagineer** [comment]
I'd like to try to temp solve this for the start of this term, so it appears correctly for users from the get-go. Can we manually set the term start and end for this Spring Session?  If so, could we set the start to correspond to the meal-plan date beginning and end dates, so the meal wheel is calibrated? Yes, that isn't exactly the class dates, but encompasses the day or so before and through exams until the meal plan ends.  Spring Dates then are Jan 28 meal plan start and May 14 meal plan ends (or May 12 when the last campus classes day).
Can we move this up to solve soon? Today the wheel is already 4-days into the term, but we are still at least two days out before meal plan and 6 days out to the start of classes. Feedback welcome.
